spec/support/evm_spec_helper.rb

```diff
@@ -51,7 +51,7 @@ class << self
   end
 
   def self.remote_guid_miq_server_zone
-    Tenant.seed
+    ActsAsTenant.default_tenant ||= Tenant.seed
```

Comment:
This memoization feels like a sporadic test failure waiting to happen.  Can we set the default tenant at the beginning of each test and reset it?